### ISSUE TITLE
Health shoud return anything different than HTTP 200 ??

### ISSUE THREAD

**Issue by toni-moreno** [toni-moreno/influxdb-srelay#2]
On  influxdb-relay

````bash
 # curl -I localhost:9096/health
HTTP/1.1 200 OK
Content-Length: 135
Content-Type: application/json
Date: Mon, 13 May 2019 09:39:41 GMT

# curl  localhost:9096/health
{"status":"healthy","healthy":{"influxdb01":"OK. Time taken 755.098µs","influxdb02":"OK. Time taken 830.941µs"}}
````

Now , no response , only HTTP 200.. 

````bash
# curl -I localhost:9096/health
HTTP/1.1 200 OK
Date: Mon, 13 May 2019 09:44:05 GMT

# curl localhost:9096/health
```

**Comment by toni-moreno** [comment]
Debug log is showing

````
2019-05-13 11:47:08 ERR Handle Health for cluster Error cluster /health not exist
2019-05-13 11:47:08 DBG ----------------------END bodyMiddleWare------------------------
2019-05-13 11:47:08 DBG ----------------------END queryMiddleWare------------------------
2019-05-13 11:47:08 ERR Handle Health for cluster Error cluster /health not exist
````
And returning HTTP 200

**Comment by toni-moreno** [comment]
Fixed in version 0.2
